dts/bindings/mtd/soc-nv-flash.yaml

```diff
@@ -17,3 +17,8 @@ properties:
      type: int
      description: address alignment required by flash write operations
      required: false
+
+    sector-size:
+     type: int
+     description: the size of one flash sector
```

Comment:
How is this different than erase-block-size?

Reply:
@mbolivar-nordic it is according to the manual (UM10914 ch 4.3.5 Flash partitions):

> The size of a sector is 32 KB and the size of a page is 256 Byte.

erase-block-size is actually the page size.
Will updating *-block-size to explain they are the page size do it?

Reply:
> @mbolivar-nordic it is according to the manual (UM10914 ch 4.3.5 Flash partitions):

If that's an SoC specific manual, I'm wondering if perhaps we need multiple compatibles for these nodes.

Reply:
sorry @mbolivar-nordic, this is an old pr and just noticed i changes a common definition. I'm refactoring a few files and pushing the changes.